pkg/hub/localizer/localizer.go

```diff
@@ -20,7 +20,12 @@ import (
 	"context"
 	"fmt"
 
+	"encoding/json"
+	"errors"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Please format your imports. Thanks.

Reply:
cannot reproduce, will recheck in newest commit